common/oidc-authservice/base/envoy-filter.yaml

```diff
@@ -10,12 +10,12 @@ spec:
     - applyTo: HTTP_FILTER
       match:
         context: GATEWAY
-      listener:
-        filterChain:
-          filter:
-            name: "envoy.http_connection_manager"
-            subFilter:
-              name: ""
+        listener:
+          filterChain:
+            filter:
+              name: "envoy.filters.network.http_connection_manager"
+              subFilter:
+                name: "envoy.filters.http.router"
```

Comment:
We can still add subFilter like 
```
              subFilter:
                name: "envoy.router"
```

Reply:
Yes, It will be correct filter also, but in runtime config i see name of that filter is envoy.filters.http.router and maybe in feature istio's release envoy.router name will stop working